src/views/blocks/rsvp/form/form.php

```diff
@@ -40,7 +43,7 @@
 		<?php if ( $must_login ) : ?>
 			<?php $this->template( 'blocks/rsvp/form/submit-login', array( 'event_id' => $event_id, 'going' => $going, 'ticket_id' => $ticket_id ) ); ?>
 		<?php else : ?>
-			<?php $this->template( 'blocks/rsvp/form/details', array( 'ticket' => $ticket ) ); ?>
+			<?php $this->template( 'blocks/rsvp/form/details', array( 'ticket' => $ticket, 'post_id' => $post_id ) ); ?>
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: e736e2767d130f01afb9f79cf0b3c604</sub>